### DISCUSSION TITLE
How can i use the generated prisma client types with Nestjs(Graphql)

### DISCUSSION THREAD

**Discussion by yassinebridi** [prisma/prisma#4737] [Q&A] [answered]
Prisma provides some really powerful arguments and input and return types, like for this Update many post(Taken from the docs):
```ts
PostUpdateManyWithoutAuthorInput
```
How can i use these kind of types directly from the prisma client inside [Nestjs graphql module](https://docs.nestjs.com/graphql/mutations#code-first)

**Comment by unlight** [comment, marked as answer]
There are some projects by community which focused on this:
https://github.com/unlight/prisma-nestjs-graphql
https://github.com/wSedlacek/prisma-generators/tree/master/libs/nestjs
https://github.com/EndyKaufman/typegraphql-prisma-nestjs

**Reply by yassinebridi** [reply]
I found your project yesterday, it's the best of them so far, it's exactly what i needed, thank you for making it available.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
